packages/rancher-cis-benchmark/charts/templates/configmap.yaml

```diff
@@ -5,7 +5,9 @@ metadata:
   namespace: {{ template "cis.namespace" . }}
 data:
   # Default ClusterScanProfiles per cluster provider type
-  rke: "rke-profile-permissive-1.6"
+  rke: |-
+    <1.16.0: rke-profile-permissive-1.5
```

Comment:
Is the space after the colon needed for something? If not we should drop it

Reply:
No, I added it for readability

Reply:
OK, that sounds fine then.